shared/src/main/scala/scodec/Err.scala

```diff
@@ -1,5 +1,7 @@
 package scodec
 
+import scala.reflect.ClassTag
```

Comment:
Is this needed?

Reply:
Removed. Would you prefer the commits to be squashed?